app/src/services/mlService.ts

```
@@ -775,8 +774,10 @@ export const useRecordingProcessor = (
   const lastProcessedTime = useSharedValue(0);
   const onLandmarksJS = createRunOnJS((lm: number[][]) => onLandmarksRef.current(lm));
   const logErrorJS = createRunOnJS(logger.error);
-  const extractLandmarksRec = useHandLandmarkExtractor();
   const pluginAvailable = isResizePluginAvailable();
+  const extractLandmarksRec = pluginAvailable
+    ? useHandLandmarkExtractor()
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)